app/models/policy_configuration.rb

```diff
@@ -16,20 +16,22 @@ class PolicyConfiguration < ApplicationRecord
     {
       routing_name: "student-loans",
       slugs: StudentLoans::SlugSequence::SLUGS,
-      policies: [StudentLoans]
+      policies: [StudentLoans],
+      i18n_namespace: "student_loans"
```

Comment:
This might be redundant/duplication as each policy model already specifies a `locale_key` e.g. https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/master/app/models/early_career_payments.rb#L36